client/lib/cart-values/index.js

```diff
@@ -374,6 +323,13 @@ export function paymentMethodName( method ) {
 	return paymentMethodsNames[ method ] || method;
 }
 
+/**
+ * Determines if a payment method is enabled
+ *
+ * @param {import('@automattic/shopping-cart').ResponseCart} cart The shopping cart
+ * @param {import('calypso/my-sites/checkout/composite-checkout/types/checkout-payment-method-slug').CheckoutPaymentMethodSlug} method The short payment method string
+ * @returns {boolean} true if enabled
+ */
 export function isPaymentMethodEnabled( cart, method ) {
 	const redirectPaymentMethods = [
```

Comment:
I seem to recall this array being moved / implemented somewhere else; maybe that is happening elsewhere in this PR chain?

Reply:
That's correct, it's https://github.com/Automattic/wp-calypso/pull/47261